modules/efk-logging-fluentd-external.adoc

```diff
@@ -17,24 +17,44 @@ ifdef::openshift-enterprise[]
 The `secure-forward` plug-in is supported by Fluentd only.
 endif::openshift-enterprise[]
 
-[NOTE]
-====
-For Rsyslog, you can edit the Rsyslog configmap to add support for Syslog log forwarding using the *omfwd* module, see link:https://www.rsyslog.com/doc/v8-stable/configuration/modules/omfwd.html[omfwd: syslog Forwarding Output Module]. To send logs to a different Rsyslog instance, you can the *omrelp* module, see link:https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html[omrelp: RELP Output Module].
-====
-
 The logging deployment provides a `secure-forward.conf` section in the Fluentd configmap
-for configuring the external aggregator:
+in the `openshift-logging` project for configuring the external aggregator:
+
+.Sample `secure-forward.conf` section
+[source,yaml]
+----
+ secure-forward.conf: |
+    # <store>
+    # @type secure_forward
```

Comment:
This sample config is for 3.11.  Use https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/configs.d/user/secure-forward.conf as the sample config for 4.x

Reply:
The sample config in the docs is what is in the 4.2 Fluentd configmap. The user should replace the default `secure-forward.conf` in the configmap with the config in the link?

Reply:
> The sample config in the docs is what is in the 4.2 Fluentd configmap

Ah, ok.  That is wrong.  We need to fix that.   https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/configs.d/user/secure-forward.conf is what should be used there instead.